stdlib/public/runtime/Casting.cpp

```diff
@@ -1289,9 +1289,8 @@ static id bridgeAnythingNonVerbatimToObjectiveC(OpaqueValue *src,
       swift_unknownObjectRetain(result);
     return result;
   }
-  
   // Dig through existential types.
-  if (auto srcExistentialTy = dyn_cast<ExistentialTypeMetadata>(srcType)) {
+  else if (auto srcExistentialTy = dyn_cast<ExistentialTypeMetadata>(srcType)) {
```

Comment:
This function has always been a long if-else chain; I've reformatted it some to make that a bit more explicit.

I've also adjusted the comment layout to hopefully make it clearer:
```
// Explain first if
if (...) {
  ...
}
// Explain second if
else if (....) {
  ...
}
// Explain third if
else if (...) {
  ...
}
```